test/verifiedTests/analysis/testFASTCORE/testLP9/testLP9.m

```diff
@@ -0,0 +1,52 @@
+% The COBRAToolbox: Lp9.m
+%
+% Purpose:
+%     - test if bug fix in LP9 affects the results for test model (without
+%     coupling constraints)
+%
+% Authors:
+%     - fix compatibility for models with coupling constraints: Agnieszka
+%     Wegrzyn 2016/03/06
+%
+
+global CBTDIR
+
+% define the features required to run the test
+requiredSolvers = {'gurobi'};
+
+% require the specified toolboxes and solvers, along with a UNIX OS
+solversPkgs = prepareTest('requiredSolvers', requiredSolvers);
+
+% save the current path and initialize the test
+currentDir = cd(fileparts(which(mfilename)));
+
+% determine the test path for references
+testPath = pwd;
+
+% load the model
+model = getDistributedModel('ecoli_core_model.mat'); %For all models in the test/models folder and subfolders
+
+% load input variables for the function
+load('options.mat');
```

Comment:
May you name this file according to the guidelines?

Reply:
I resolved it by adding all the referenceData and input variables into one file "testData_LP9.mat"